articles/sso/v2/single-page-apps-sso.md

```diff
@@ -0,0 +1,177 @@
+---
+description: How to implement client-side SSO on single page applications.
+toc: true
+---
+
+# Client-Side SSO on Single Page Applications
+
+Single Page Applications (SPA) are user-friendly apps that load a single HTML page. This page then dynamically updates as the users interacts with the app. If your SPA is associated with other apps or sites that asks for authentication, you can implement OIDC-compliant Single Sign On to minimize the number of times the user has to provide their credentials. This ensures both the security of the process and ease of use from the perspective of the user.
+
+## What SSO Looks Like
+
+Suppose you have three applications:
+
+* App 1: app1.com (single page app)
+* App 2: app2.com (single page app)
+* App 3: app3.com (regular web app)
+
+If you've implemented SSO and the user logs in to *any* of the three applications, the user should automatically be logged in to the other applications.
+
+This article shows you how to implement OIDC-compliant Single Sign On via [Silent Authentication](/api-auth/tutorials/silent-authentication) for your Single Page Applications using [this sample](https://github.com/auth0-samples/oidc-sso-sample). This process involves
+
+## Implement SSO on Your SPA
+
+:::panel-warning Prerequisites
+This article assumes that you have already created an Auth0 Client (of type **Single Page Web Applications**) with the **OIDC Conformant** flag enabled.
+:::
+
+### Get Started with the OIDC-Compliant Single Sign On Sample
+
+Please feel free to [download the sample](https://github.com/auth0-samples/oidc-sso-sample) and work through the examples on your local environment as you read this doc. The following instructions will help you set up the sample SPA on your local environment.
+
+<div class="alert alert-info">
+  <strong>Heads up!</strong> This document assumes that you're using port 3000 when running the sample. If you are using a different port, you'll need to adjust for this as you work through the sample (specifically the <i>auth0-variables.js</i>, <i>callback.html</i>, and <i>index.js</i> files) and configure your Auth0 Client.
+</div>
+
+1. Add `http://localhost:3000` and `http://localhost:3000/callback.html` to the Allowed Callback URLs field of your [Auth0 Client Settings](${manage_url}/#/clients/${account.clientId}/settings).
```

Comment:
Missing first bullet: Create new Client (and select SPA). You can add it in the same bullet as well, I don't mind, just don't jump directly to Allowed Callback URLs.

I know you have a panel earlier, but I would remove the panel and add one extra bullet: create a client, select SPA, set the OIDC Conformant toggle to true.